demo/src/main/java/com/basgeekball/awesomevalidation/demo/DemoActivity.java

```diff
@@ -168,6 +155,17 @@ private void addValidationForEditText(Activity activity) {
         mAwesomeValidation.addValidation(activity, R.id.edt_zipcode, "\\d+", R.string.err_zipcode);
         mAwesomeValidation.addValidation(activity, R.id.edt_year, Range.closed(1900, Calendar.getInstance().get(Calendar.YEAR)), R.string.err_year);
         mAwesomeValidation.addValidation(activity, R.id.edt_height, Range.closed(0.0f, 2.72f), R.string.err_height);
+        mAwesomeValidation.addValidation(activity, R.id.edt_birth, new CustomValidation() {
+            @Override
+            public boolean compare(String input) {
+                try {
+                    new SimpleDateFormat("dd/MM/yyyy", Locale.US).parse(input);
```

Comment:
Same as previously, could you please make a slightly complicated example here, for instance: check if the age > 18?  Thanks.

Reply:
Same.